### PR TITLE
New RUM Performance Monitoring summary page

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do? What is the motivation?

- Updates the RUM landing page image with the updated Performance Summary page
- Adds info about the new Performance Summary page

### Merge instructions

Merge readiness:
- [ ] Ready for merge

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### Additional notes

Requires review/approval from product.

### CODE REVIEW

File: content/en/real_user_monitoring/_index.md [hunk 1]
```diff
@@ -126,8 +126,26 @@ All Datadog SDKs traffic is transmitted over SSL (default 443) to the following
 
 Access RUM by navigating to [**Digital Experience > Performance Summary**][1].
 
+Select an application from the top navigation, or follow the setup instructions for [browser][15] or [mobile][16] to add your first one.
```
Comment (neko-dd) [inline content/en/real_user_monitoring/_index.md:129]: your first one what? application?

File: content/en/real_user_monitoring/_index.md [hunk 2]
```diff
@@ -126,8 +126,26 @@ All Datadog SDKs traffic is transmitted over SSL (default 443) to the following
...
+
+{{< img src="real_user_monitoring/rum-performance-application-selector.png" alt="Select a RUM application" >}}
+
 **Tip**: To open RUM from Datadog's global search, press <kbd>Cmd</kbd>/<kbd>Ctrl</kbd> + <kbd>K</kbd> and search for `real user monitoring`.
 
+## Performance monitoring summary
+
+{{< img src="real_user_monitoring/rum-performance-browser.png" alt="RUM Performance monitoring summary page for a browser application." >}}
```
Comment (neko-dd) [inline content/en/real_user_monitoring/_index.md:137]:
```suggestion
{{< img src="real_user_monitoring/rum-performance-browser.png" alt="RUM Performance Monitoring summary page for a browser application" >}}
```

File: content/en/real_user_monitoring/_index.md [hunk 3]
```diff
@@ -126,8 +126,26 @@ All Datadog SDKs traffic is transmitted over SSL (default 443) to the following
...
+
+The [RUM Performance Monitoring summary][1] page, which provides relevant and actionable insights for both web and mobile applications. You have a tailored experience for each platform that helps you:
```
Comment (neko-dd) [inline content/en/real_user_monitoring/_index.md:139]:
```suggestion
The [RUM Performance Monitoring summary][1] page provides relevant and actionable insights for both web and mobile applications. You have a tailored experience for each platform that helps you:
```

File: content/en/real_user_monitoring/_index.md [hunk 4]
```diff
@@ -126,8 +126,26 @@ All Datadog SDKs traffic is transmitted over SSL (default 443) to the following
...
+
+- **Focus on key metrics** by platform, such as the UI latency for web or mobile crashes
+- **Monitor application health** through familiar KPIs, such as Core Web Vitals for web apps or hang rate for iOS to assess app reliability
```
Comment (neko-dd) [inline content/en/real_user_monitoring/_index.md:142]:
```suggestion
- **Monitor application health** through familiar KPIs, such as Core Web Vitals for web apps or hang rate for iOS, to assess app reliability
```

File: content/en/real_user_monitoring/_index.md [hunk 5]
```diff
@@ -126,8 +126,26 @@ All Datadog SDKs traffic is transmitted over SSL (default 443) to the following
...
+- **Dive into investigations directly** from interactive widgets without leaving the page
+
+For **web apps**, use the search bar to filter data, identify slow pages, and follow the UI to the [/rum/optimization/inspect][17] page.
```
Comment (neko-dd) [inline content/en/real_user_monitoring/_index.md:145]:
```suggestion
For **web apps**, use the search bar to filter data, identify slow pages, and follow the UI to the [RUM Optimization Inspect][17] page.
```

File: content/en/real_user_monitoring/_index.md [hunk 6]
```diff
@@ -126,8 +126,26 @@ All Datadog SDKs traffic is transmitted over SSL (default 443) to the following
 
 Access RUM by navigating to [**Digital Experience > Performance Summary**][1].
 
+Select an application from the top navigation, or follow the setup instructions for [browser][15] or [mobile][16] to add your first application.
+
+{{< img src="real_user_monitoring/rum-performance-application-selector.png" alt="Select a RUM application" >}}
+
 **Tip**: To open RUM from Datadog's global search, press <kbd>Cmd</kbd>/<kbd>Ctrl</kbd> + <kbd>K</kbd> and search for `real user monitoring`.
 
+## Performance monitoring summary
```
Comment (MaelNamNam) [inline content/en/real_user_monitoring/_index.md:135]: I like the content, but I feel we're again missing something that shows what the mobile version looks like. Could we put two screenshots side by side, one for web, one for mobile, please?

Note: to take a good screenshot of a mobile app for the Demo org, you might want to deselect `user` from the Session Type filter as almost all of the traffic actually comes from Synthetics.

File: content/en/real_user_monitoring/_index.md [hunk 7]
```diff
@@ -126,8 +126,26 @@ All Datadog SDKs traffic is transmitted over SSL (default 443) to the following
...
+
+{{< img src="real_user_monitoring/rum-performance-browser.png" alt="RUM Performance Monitoring summary page for a browser application" >}}
+
+The [RUM Performance Monitoring summary][1] page provides relevant and actionable insights for both web and mobile applications. You have a tailored experience for each platform that helps you:
+
+- **Focus on key metrics** by platform, such as the UI latency for web or mobile crashes
```
Comment (MaelNamNam) [inline content/en/real_user_monitoring/_index.md:141]:
```suggestion
- **Focus on key data points** by platform, such as the UI latency for web or mobile crashes
```

Those are currently event based. They will be metric based in the future with RUM Tail-Based, but I'd rather use a generic term

**Review by MaelNamNam — CHANGES_REQUESTED** [review]
@rtrieu I left a comment.

But also: could you update the screenshot on `/real_user_monitoring/browser/` please?

File: content/en/real_user_monitoring/_index.md (hunk 6 above)
Comment (rtrieu) [reply content/en/real_user_monitoring/_index.md:135]: Done - thanks for the suggestion!

**Review by MaelNamNam — APPROVED** [review]
LGTM

**Pull request merged**